generator/views/glean_ping_view.py

```diff
@@ -216,16 +216,12 @@ def _get_category_and_name(self, metric: GleanProbe) -> Tuple[str, str]:
 
         return category, name
 
-    def _to_looker_name(self, metric: GleanProbe, suffix: str = "") -> str:
+    def _to_looker_name(self, metric: GleanProbe) -> str:
```

Comment:
This function is not used by the dimension names, only by the view names of the joined labeled counters. Options are to either use this in `_make_dimension`, or update that code as well.